novatel_gps_driver/src/nodelets/novatel_gps_nodelet.cpp

```diff
@@ -490,10 +497,6 @@ namespace novatel_gps_driver
           gps_.SetImuRate(imu_sample_rate_, true);
         }
       }
-      if (publish_novatel_velocity_)
```

Comment:
> @pjreed the CI seems to have kicked the bucket and I'm not sure how to re-run it.
> 
> The build actually succeeded but it didn't make it to this page. https://travis-ci.org/github/swri-robotics/novatel_gps_driver/builds/696632167

I just updated the repo to use GitHub Actions instead of Travis, so try merging master into this and pushing it to see if it re-runs it now.